src/schemas/json/jekyll.json

```diff
@@ -66,9 +66,10 @@
         "/:path",
         "/:name",
         "/:title",
-        "/:output_ext"
+        "/:output_ext",
+        "/blog/:title"
       ],
-      "pattern": "^(/(:(collection|path|name|title|output_ext))+)+$"
+      "pattern": "^(/(.*/)?(:(collection|path|name|title|output_ext))+)+$"
```

Comment:
I interpret the bug as - any path in the URL can contain some string that is not `:(collection|path|name|title|output_ext)` - so not just the start. I think that this is so lenient, that it may not be worth having a `pattern`. What do you think?